stories/molecules/components/footer/Footer.stories.mdx

```diff
@@ -0,0 +1,189 @@
+import { ArgsTable, Meta, Story, Canvas, Preview, Props } from '@storybook/addon-docs'
+import { ThemeProvider } from 'ui/atoms/theme/ThemeProvider'
+import { ThemeSwitcher } from 'ui/atoms/theme/ThemeSwitcher'
+import { Footer } from 'ui/molecules/footer/Footer'
+import { Typography } from 'ui/atoms/typography/Typography'
+import { loadTranslations } from 'i18n/utils'
+import { useTranslation } from 'hook/useTranslation'
+import footer_en from './i18n/footer_en.json'
+import footer_fr from './i18n/footer_fr.json'
+import footer_se from './i18n/footer_se.json'
+import footer_it from './i18n/footer_it.json'
+import { Button } from 'ui/atoms/button/Button'
+
+<Meta
+  title="Molecules/Footer"
+  component={Footer}
+  parameters={{
+    actions: { argTypesRegex: '^on.*' },
+    docs: {
+      source: {
+        type: 'code',
+        format: true,
+        language: 'jsx'
+      }
+    }
+  }}
+  decorators={[
+    Story => {
+      return (
+        <ThemeProvider>
+          <div style={{ padding: '20px 30px' }}>
+            <ThemeSwitcher />
+          </div>
+          <Story />
+        </ThemeProvider>
+      )
+    }
+  ]}
+/>
+
+# Footer
+
+> Helps users to find useful information and site functionality.
+
+[![stability-unstable](https://img.shields.io/badge/stability-unstable-yellow.svg)](https://github.com/emersion/stability-badges#unstable)
+
+## Description
+
+**Footer** is a responsive component which embeds two main elements:
+
+- A `LanguageSwitcher` component that offers users the ability to change the language of the application.
+- A `secondaryElement` Optional element that is positioned last in the reading flow.
+
+## Overview
+
+<Story
+  name="Overview"
+  argTypes={{
+    languages: {
+      control: false,
+      table: {
+        type: { summary: 'Languages' }
+      }
+    },
+    secondaryElement: {
+      control: false,
+      table: {
+        type: { summary: 'JSX.Element' }
+      }
+    }
+  }}
+  args={{
+    languages: [
+      {
+        name: 'English',
+        lng: 'en'
+      },
+      {
+        name: 'Français',
+        lng: 'fr'
+      }
+    ]
+  }}
+>
+  {args => {
+    const translationsToLoad = [
+      { lng: 'en', namespace: 'footer', resource: footer_en },
+      { lng: 'se', namespace: 'languageSwitcher', resource: footer_se },
+      { lng: 'it', namespace: 'languageSwitcher', resource: footer_it },
+      { lng: 'se', namespace: 'footer', resource: footer_se },
+      { lng: 'it', namespace: 'footer', resource: footer_it },
+      { lng: 'fr', namespace: 'footer', resource: footer_fr }
+    ]
+    loadTranslations(translationsToLoad)
```

Comment:
Maybe we could refactor all code associated to translations into a dedicated function.

Reply:
Yes it would improve readability, will do 😃 👍